docs/source/api_commands.rst

```diff
@@ -559,9 +674,19 @@ Parameters:
 get_serial_number
 -----------------
 
-Description:
+Description: Get serial number of the vacuum.
 
-Parameters:
+Parameters: None
+
+Returns:
+
+    serial_number: Serial number of the vacuum.
+
+Return example::
+
+    {'serial_number': 'B16EVD12345678'}
```

Comment:
If this is yours, I would change it and make it generic

Reply:
Yeah I did modify it, but appreciate the back up! Wouldn't be the first time I forgot to obfuscate something. 😄